ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-postsubmits.yaml

```
@@ -145,6 +145,52 @@ postsubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    cluster: build04
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+      timeout: 5h0m0s
+    labels:
+      ci.openshift.io/generator: prowgen
+    max_concurrency: 1
```

```suggestion
    max_concurrency: 6
```
as @gavin-stackrox suggested.

Compare with the value for `branch-ci-stackrox-stackrox-master-merge-gke-ui-e2e-tests`.